--- nexus-master/lib/nexus_web/router.ex	2026-06-21 19:39:28.000000000 +0000
+++ stage3/lib/nexus_web/router.ex	2026-07-17 04:56:45.547489486 +0000
@@ -113,6 +113,7 @@
   # API v1 — public read
   scope "/api/v1", NexusWeb.API.V1 do
     pipe_through :api
+    get "/boot",                   BootController,   :index
     get "/spaces",                 SpaceController,  :index
     get "/themes",                 ThemeController,  :index
     get "/spaces/:slug",           SpaceController,  :show
